### DISCUSSION TITLE
Could you supply built-in  CharacterController ?

### DISCUSSION THREAD

**Discussion by finscn** [jrouwe/JoltPhysics.js#19] [General]
There are  CharacterController in PhysX , bullet,  ammo .
But I've not find it in Jolt. 

Could you supply built-in  CharacterController ?

If not , could you tell me why or  tell me which APIs can help me to implement CharacterController ?

**Comment by finscn** [comment]
I found  https://jrouwe.github.io/JoltPhysics/class_character_virtual.html 
But  it's not in  JoltPhysics.js  (missing in  `.d.ts`)

**Reply by jrouwe** [reply]
A lot of functionality from JoltPhysics has not been implemented for JoltPhysics.js yet (that's why the library is at version 0.0.X).

Feel free to add CharacterVirtual to https://github.com/jrouwe/JoltPhysics.js/blob/main/JoltJS.idl and send me a PR.

**Reply by finscn** [reply]
My ability is not enough to create a pull-request for Jolt , so sorry.  

I can only create a feature-request  in issues page .

**Comment by jrouwe** [comment]
For future reference, this is supported now, see: https://jrouwe.github.io/JoltPhysics.js/character_virtual.html
